### PR TITLE
Filter alerts and logs by timestamp

### PR DESCRIPTION
**Pull request by juliamagan**
|Related issue|
|:--:|
| #5108 |

## Description

- We have fixed a Grafana package wrong with the expected version.
- Added not to ignore timeout errors in waiters.
- Fixed a regex
- Removed extra steps in the setup

---

## Testing performed

| Validation | Jenkins | Local   | Commit | Notes                |
|--------------------|-----------|---------|--------|--------|
|    `end_to_end/test_vulnerability_detector`       | [🟡 ](https://ci.wazuh.info/view/Tests/job/Test_e2e_system/248/) | [🟡 ](https://github.com/wazuh/wazuh-qa/files/14836599/test_new.zip) |      225bb88         | Known issue: https://github.com/wazuh/wazuh-qa/issues/5173 and https://github.com/wazuh/wazuh-qa/issues/5175 |

### CODE REVIEW

**Review by rafabailon — APPROVED** [review]
LGTM

File: deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py [hunk 1]
```diff
@@ -167,15 +167,14 @@ def install_package(host: str, operation_data: Dict[str, Dict], host_manager: Ho
...
+
                 package_data = load_packages_metadata()[package_id]
                 package_url = package_data['urls'][host_os_name][host_os_arch]
 
                 logging.info(f"Installing package on {host}")
                 logging.info(f"Package URL: {package_url}")
 
-                current_datetime = datetime.utcnow().isoformat()
-
+                current_datetime = datetime.now(timezone.utc).isoformat()[:-6]
```
Comment (Rebits) [inline deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py:177]: Add a comment about why `[:-6]`

**Review by Rebits — CHANGES_REQUESTED** [review]
Great job! Some minor changes are requested

File: deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py (hunk 1 above)
Comment (juliamagan) [reply deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py]: Done in c0e9601

Comment (Rebits) [reply deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py]: LGTM

**Pull request merged**